### DISCUSSION TITLE
Features for 09_theta_image_save

### DISCUSSION THREAD

**Discussion by codetricity** [caitlynbakery/theta_image_save_9#1] [Ideas]
Change name of project to 09_theta_image_save as it will make it easier to see the sequence.

Per our discussion, there are no images displayed on 09_theta_image_save.

![image](https://user-images.githubusercontent.com/5799860/178527843-9f27ceb7-0bf2-4274-8a7c-21099d9e3cb4.png)

There are two buttons.  The first button gets 5 images from lorem picsum and saves it to local storage on the mobile phone.  As there are many different places to store local storage, focus on saving it to the image or photos directory of an Android device.

The second button repeats the process with the images on the RICOH THETA camera.

Test that the images were saved locally on the mobile device by using the built-in Android Gallery app

<img width="191" alt="image" src="https://user-images.githubusercontent.com/5799860/178530691-c671049d-430f-463c-8b68-773b59b3d402.png">

**Comment by caitlynbakery** [comment]
Ok, I'll change the name of the project and will try saving the image to local storage.

**Comment by codetricity** [comment]
prior to going to concept 10,

1. get adb working on MacBook from command line - you need to add adb to your path on the Mac.  research on Google on to do this
2. use `adb shell` on the Mac when the Android emulator is running to ssh into the emulator and go through the directory structure of the emulator.  research on Google where files are stored on Android
3. use your oppkey email account to create a google play account to update the photos app
4. run the android emulator with software rendering only (not hardware) as a test
5. get an iOS emulator running on the MacBook
6. configure project to use insecure http on the iOS emulator - search on Google for Flutter insecure http API calls on iOS simulator configuration

**Reply by caitlynbakery** [reply]
Ok, I updated the adb shell and found out where the files are being stored.

Here is the screenshot of where the files are stored on my emulator:

![Screen Shot 2022-07-14 at 7 57 33 AM](https://user-images.githubusercontent.com/22283010/179013797-962c09af-2e1a-434a-b591-48c7ac7c3922.png)

**Comment by codetricity** [comment]
I've tried different emulators on Windows 11.  All showed problems with Google Photos and RICOH THETA images.  The Google Photos 360 view works phone on a physical device Pixel 2, which I I can loan to @caitlynbakery for testing.

see information below.

https://youtu.be/vv5uX9oa_gE

**Reply by caitlynbakery** [reply]
I tested the app with Google Pixel 2 and the Google Photos worked without any problems. I also installed Vysor. :)
